src/renderers/shared/utils/ReactErrorUtils.js

```diff
@@ -142,7 +142,11 @@ if (__DEV__) {
      * real browser event.
      */
     const fakeNode = document.createElement('react');
-    let depth = 0;
+
+    const REACT_DEV_ERROR_DEPTH = '__reactDevErrorDepth';
+    if (global[REACT_DEV_ERROR_DEPTH] === undefined) {
```

Comment:
I don't think so. Another problem is that this means that React cannot work in frozen Realms.